src/charm.py

```diff
@@ -401,14 +434,24 @@ def _on_remove(self, event: RemoveEvent) -> None:
     def _on_kratos_relation_changed(self, event: RelationChangedEvent) -> None:
         self._handle_status_update_config(event)
 
+    def _on_oathkeeper_info_relation_changed(
+        self, event: OathkeeperInfoRelationChangedEvent
+    ) -> None:
+        self._update_oathkeeper_info_relation_data(event)
```

Comment:
Why is this call needed here?

Reply:
Oathkeeper needs to send its data when the info relation joins